File: leetcode/1470.重新排列数组.cpp
#include <iostream>
#include <vector>
using namespace std;
/*
 * @lc app=leetcode.cn id=1470 lang=cpp
 *
 * [1470] 重新排列数组
 */

// @lc code=start
class Solution
{
public:
    vector<int> shuffle(vector<int> &nums, int n)
    {
        vector<int> res;
        for (size_t i = 0; i < n; i++)
        {
            res.push_back(nums[i]);
            res.push_back(nums[i + n]);
        }
        return res;
    }
};
// @lc code=end

File: leetcode/1470.重新排列数组.py
#
# @lc app=leetcode.cn id=1470 lang=python3
#
# [1470] 重新排列数组
#
from typing import List


# @lc code=start
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        return [
            num
            for i in range(n)
            for num in (nums[i], nums[i+n])
        ]
# @lc code=end
